vllm/model_executor/model_loader/loader.py

```diff
@@ -1257,9 +1257,24 @@ def _get_gguf_weights_map(self, model_config: ModelConfig):
         """
         config = model_config.hf_config
         model_type = config.model_type
+        gguf_to_hf_name_map = {}
         # hack: ggufs have a different name than transformers
         if model_type == "cohere":
             model_type = "command-r"
+        if model_type == "deepseek_v3":
```

Comment:
```suggestion
        if model_type in ("deepseek_v3", "deepseek_v2"):
```
I think we can also support deepseek-v2 gguf model in this PR. This would allow us to evaluate deepseek gguf model with deepseek-v2-lite instead of R1/V3: https://huggingface.co/legraphista/DeepSeek-V2-Lite-Chat-IMat-GGUF?show_file_info=DeepSeek-V2-Lite-Chat.IQ4_XS.gguf